### DISCUSSION TITLE
Please open issue 10615

### DISCUSSION THREAD

**Discussion by AndersBennedsgaard** [kubeflow/pipelines#10753] [General]
https://github.com/kubeflow/pipelines/issues/10615 was opened to stop the project from using deprecated container registries, but was (wrongly) closed in favor of creating a discussion about it here. I think it should be re-opened such that we can get it fixed.

(since this message isn't a bug/feature report for the project, I've elected to open a discussion about this instead of an issue)

**Comment by HumairAK** [comment]
@AndersBennedsgaard agree, I reopened it

**Comment by AndersBennedsgaard** [comment]
Perfect, thanks :+1:
